service/src/test/resources/data/mocked_request_data_details.txt

```diff
@@ -1,5 +1,7 @@
 identifier	filepath	description
 mockIncomingRequest1JsonDataWith2T2N	incoming_requests/mocked_request1_complete_tumor_normal.json	Mock incoming request json with 2 tumor samples and 2 matching normals
+mockIncomingRequest1UpdatedJsonDataWith2T2N	incoming_requests/mocked_request1_updated_complete_tumor_normal.json	Mock incoming request json with 2 tumor samples and 2 matching normals with some metadata field updates.
+mockIncomingRequest1UpdatedSampleJsonDataWith2T2N	incoming_requests/mocked_request1_updated_sample_complete_tumor_normal.json	Mock incoming request json with 2 tumor samples and 2 matching normals with some sample metadata field updates.
```

Comment:
once you consolidate the sample updates into the other updated mock request json file then you can delete the entry here